cmd/oras/internal/option/pusher.go

```diff
@@ -73,3 +98,22 @@ func decodeJSON(filename string, v interface{}) error {
 	defer file.Close()
 	return json.NewDecoder(file).Decode(v)
 }
+
+// getAnnotationsMap resharps annotationslice to k-v type and updates annotations
+func getAnnotationsMap(ManifestAnnotations []string, annotations map[string]map[string]string) error {
+	re := regexp.MustCompile(`=\s*`)
+	rawAnnotationsMap := make(map[string]string)
+	for _, rawAnnotationStr := range ManifestAnnotations {
```

Comment:
short and concise names are idiomatic in golang.
```suggestion
	for _, anno:= range ManifestAnnotations {
```

Reply:
updated